src/core/annotation.js

```diff
@@ -628,8 +628,11 @@ class Annotation {
    * @private
    */
   _isPrintable(flags) {
+    // In Acrobat, hidden flag cancels the print one
+    // (see annotation_hidden_print.pdf).
```

Comment:
*Just to be sure:* Have you checked that this indeed applies to *all* annotation-types, and not just the Widget ones?

Reply:
Yes and I just added a test to make sure we don't have any regressions in the future.